### ISSUE TITLE
[bug] Not idempotent - ports with profiles

### ISSUE THREAD

**Issue by GeoffWilliams** [GeoffWilliams/puppet-windows_firewall#2]
Testcase causes flip-flopping - investigating
```
windows_firewall_rule { "puppet - open port in specific profiles":
  ensure    => present,
  direction => "in",
  action    => "allow",
  protocol  => "tcp",
  profiles  => ["private", "domain"],
  localport => "666",
}
```

**Comment by GeoffWilliams** [comment]
its to do with the order of the `profiles` array - needs to be the same for comparison

**Comment by GeoffWilliams** [comment]
fixed in 0.2.0
